web_src/css/features/projects.css

```diff
@@ -98,3 +98,27 @@
 .card-ghost * {
   opacity: 0;
 }
+
+.fullscreen.projects-view .ui.container.flex-text-block {
```

Comment:
Don't use such trick. Why not use a clear CSS class name for the target elements?